plugins/logging/pkg/errors/errors.go

```diff
@@ -11,6 +11,7 @@ import (
 var (
 	ErrInvalidList             = utilerrors.New(codes.FailedPrecondition, errors.New("list did not return exactly 1 result"))
 	ErrInvalidPersistence      = utilerrors.New(codes.InvalidArgument, errors.New("invalid persistence config"))
+	ErrInvalidDataPersistence  = utilerrors.New(codes.InvalidArgument, errors.New("minimum of 2 data nodes required if no persistent storage"))
```

Comment:
Should this message match the other one?

Reply:
Nope, this is technically a different error state.  The first one is for if there is a mismatch or misconfiguration in the actual persistence settings.  In this error the settings are technically valid (or non existent) but lead to an unsafe cluster configuration.

Reply:
Sorry I was referring to the wording between this error and the other log message in this PR